app/views/layouts/application.html.erb

```diff
@@ -21,8 +21,8 @@
 
     <%= build_plugin_html 'server:before-script-load' %>
 
-    <link rel="preload" href="<%= script_asset_path "start-discourse" %>" as="script">
-    <link rel="preload" href="<%= script_asset_path "browser-update" %>" as="script">
+    <% add_resource_preload_list(script_asset_path("start-discourse"), "script") %>
+    <% add_resource_preload_list(script_asset_path("browser-update"), "script") %>
```

Comment:
On browsers that do not yet support early hints, do we lose the preload with the link tag?

Reply:
No, we do not as they are interchangeable

Reply:
O nice, I didn't know about this.